packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1763,6 +1790,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
      * @param options - options controlling how the summary is generated or submitted
      */
     public async submitSummary(options: ISubmitSummaryOptions): Promise<SubmitSummaryResult> {
+        if (this.createContainerMetadata.summaryCount !== undefined) {
```

Comment:
Should it not be undefined? And initiated when container runtime instantiates (!existing, plus filling it with zero for old files that do not have it)?
Otherwise we never start counting for existing files. Maybe it's fine, but it feels like we do want to start counting them :)

Reply:
yeah it's fine either way